run_scans.py

```diff
@@ -134,22 +146,29 @@ def launchTestsArgs(tool, slot, link, chamber, scanmin, scanmax, nevts,
 
   from qcoptions import parser
 
-  parser.add_option("--series", action="store_true", dest="series",
```

Comment:
why are you removing this option? we should be able to run the scans if series if we want...